app/interfaces/api/v1/dropdown_data.rb

```diff
@@ -50,13 +50,16 @@ def role
           desc 'Return all Offence-ids to be used in advocate claims (see OffenceClasses for Litigator claims).'
           params do
             optional :offence_description, type: String, desc: 'Offences matching description'
+            optional :rep_order_date,
+                     type: String,
+                     desc: 'OPTIONAL: Date of representation order in YYYY-MM-DD',
+                     standard_json_format: true
           end
           get do
-            offences = if params[:offence_description].present?
-                         Offence.where(description: params[:offence_description])
-                       else
-                         Offence.all
-                       end
+            scheme_date = params[:rep_order_date] || '2018-01-01'
```

Comment:
I'd be tempted to put a scheme selector option on the grape documentation here, either in addition or instead of the date sensitive option above

e.g. 
```
params :offence_filter do
  optional :offence_scheme, type: String, desc: I18n.t('api.v1.dropdown_data.params.scheme_filter'), values: %w[9 10]
end
```

and use


```
# ineffecient
offences.select { |o| o.fee_schemes.first.number.eql?(offence_scheme) }
```